### ISSUE TITLE
Active Claims detail | Estimated decision date Accessibility - Not readable with TalkBack on Android

### ISSUE THREAD

**Issue by digitalken** [department-of-veterans-affairs/va.gov-team#17476]
## What happened?
Android TalkBack - Unable to single out this portion of the screen (Estimated decision date) for Android to read.  iOS is no problem.
NOTE: The Review Locations button in this block is being read as "The button button, double-tap to active".  Should probably say "Review Locations, button double-tap to activate".

## Specs:
- Device: Motorola Moto G7, Samsung Galaxy S7, latest OS.

## Steps to Reproduce
- With TalkBack enabled, attempt to read the Estimated decision date portion of the screen.

## Desired behavior
All parts of the screen are read with screen reader software.

## Acceptance Criteria
- [ ] All of the Estimated decision date portion of claims detail is read.

## Bug Severity
- 4 - Low

## Linked to Story
https://app.zenhub.com/workspaces/va-mobile-app-5f205eac3155a2000f422ecb/issues/department-of-veterans-affairs/va.gov-team/16777

## Screen shot(s) and additional information

![image.png](https://images.zenhubusercontent.com/5f6a83c84093730d05b5cee6/ac7e932e-9ccc-4fcb-9ef4-cbc1cf85835c)

**Comment by digitalken** [comment]
PASS 1/12/2012 - All items on screen read by Android.
